pyproject.toml

```
@@ -65,7 +65,7 @@ commands =
     pip install -U pip
     pip --version
     pip freeze
-    coverage run -a runtests.py
+    coverage run -a runtests.py --failfast  --tag=xxx
```

can you remove the fail fast and tag